CODEOWNERS

```diff
@@ -1 +1,2 @@
 * @2007-Instructors
+my branch!!!
```

Comment:
Any specific reasons why you're adding this to code owners? This file should generally be controlled by us since It allows us to set up behind the scenes stuffs

Reply:
Yeah, this file can't be changed - we'll have to block the review until this change is removed.

Reply:
haha, i added this code when I pushed my first branch to repo....just to test it out I was able to create my branch